tests/test__utils.py

```diff
@@ -1,77 +1,99 @@
 import functools
+from typing import Any, Callable, Coroutine, cast
 
 from starlette._utils import is_async_callable
 
 
-def test_async_func():
-    async def async_func():
+def test_async_func() -> None:
+    async def async_func() -> None:
         ...  # pragma: no cover
 
-    def func():
+    def func() -> None:
         ...  # pragma: no cover
 
     assert is_async_callable(async_func)
     assert not is_async_callable(func)
 
 
-def test_async_partial():
-    async def async_func(a, b):
+def test_async_partial() -> None:
+    async def async_func(a: Any, b: Any) -> None:
         ...  # pragma: no cover
 
-    def func(a, b):
+    def func(a: Any, b: Any) -> None:
         ...  # pragma: no cover
 
     partial = functools.partial(async_func, 1)
     assert is_async_callable(partial)
 
-    partial = functools.partial(func, 1)
+    partial = functools.partial(
+        cast(
+            Callable[..., Coroutine[Any, Any, None]], func
+        ),
+        1,
+    )
```

Comment:
What's the error here? Can we just have a type ignore?

Reply:
errors:
`tests/test__utils.py:78: error: Argument 1 to "partial" has incompatible type "Sync"; expected "Callable[..., Coroutine[Any, Any, None]]"  [arg-type]`

`tests/test__utils.py:78: note: "Sync.__call__" has type "Callable[[Arg(Any, 'a'), Arg(Any, 'b')], None]"`

Yes, we can just ignore the type here. Thank you!